scripts/MultiTester.js

```diff
@@ -26,7 +39,22 @@ class MultiTester extends RuleTester {
 
 	run(name, rule, tests) {
 		Object.entries(this._liferay.parsers).forEach(([key, parser]) => {
-			parser.run(`${name} (parser: ${key})`, rule, tests);
+			// Not all tests can run on all parsers, so we filter first.
+			const handleSkips = test => {
+				const {skip, ...config} = test;
```

Comment:
It is no enough to just read `skip` from the conf; we have to prune it out, otherwise ESLint will yell and scream about an unrecognized property.